### ISSUE TITLE
Documentation for new Wallets (part 1)

### ISSUE THREAD

**Issue by ipopescu** [casper-network/docs#298]
We need to add documentation for wallets, which will contain:
- A token workflow with RPC and a single signature.

Create a new workflow using **make-transfer and send-deploy**. This is a new workflow that can be used when creating a wallet where we need to first prepare the transfer (a.k.a. deploy), then decide who needs to sign it, and only after sending the transfer/deploy.

Add this page as part of the "How To's": https://docs.casperlabs.io/en/latest/workflow/index.html
and
https://casper.network/docs/workflow

Work with @George-cl, @darthsiroftardis, and with the ecosystem team to understand this topic.

**Comment by GloreenS** [comment]
Session to understand the topics, completed with @zie1ony, documentation is in progress.

**Comment by GloreenS** [comment]
The make-transfer command described in the [call](https://casperlabs.zoom.us/rec/share/lJw7TMHnas8Xg8hcjWVkHMCiNq-uJHdqkLuSPhQ-yb-C6ImcfYedevOR8stIx6Ye.f1VZRml3ZRZyAdnQ) Passcode: *7wW!f#? is not working as expected.
Created accounts on testnet and funded the accounts to execute the transfer. The transfer command as described in https://docs.casperlabs.io/en/latest/workflow/transfer-workflow.html?highlight=make%20transfer#transfer is working correctly with the same keys.
Task put on hold until 27 Mon, for further troubleshooting.

**Comment by ipopescu** [comment]
Time boxed for this sprint. Closing and opening the next part.

**Comment by GloreenS** [comment]
Received clarification for the commands from Karan. Explained the commands and issues to Iulia before closing this issue.
